packages/server/src/tests/globalSetup.ts

```diff
@@ -0,0 +1,26 @@
+import { GenericContainer, Wait } from "testcontainers"
+
+export default async function setup() {
+  const container = await new GenericContainer("budibase/couchdb")
+    .withExposedPorts(5984)
+    .withEnvironment({
+      COUCHDB_PASSWORD: "budibase",
```

Comment:
Should we use the `process.env` to always align with the `jestEnv.ts`?

Reply:
I believe this runs _before_ `jestEnv.ts` runs, so I'm not sure `process.env` will contain values for this yet. I can try tomorrow, though. It would be nice to have a single source of truth for these.

Reply:
Yeah, sadly `jestEnv.ts` runs after `globalSetup.ts` so there's no `COUCHDB_*` variables in there yet. Going to leave this as is, unless you can think of a way to do this.

Reply:
I'm trying to check this, but the parameter names in the jest config would suggest that the env comes before the setup files:
![image](https://github.com/Budibase/budibase/assets/4407001/d370bef8-029a-4827-b502-a0379ddf9242)